src/js/helpers/disasterHelper.js

```diff
@@ -33,6 +33,20 @@ export const fetchAgencyCount = (params) => apiRequest({
     data: params
 });
 
+export const fetchAwardSpendingByAgency = (params) => apiRequest({
+    isMocked: true,
+    url: 'v2/disaster/agency/spending/',
+    method: 'post',
+    data: params
+});
+
+export const fetchFaceValueOfLoans = (params) => apiRequest({
```

Comment:
Could we include "agency" in the name of this function? We'll be fetching loan data for multiple sections